arkouda/pdarrayIO.py

```diff
@@ -581,3 +588,144 @@ def save_all(columns : Union[Mapping[str,pdarray],List[pdarray]], prefix_path :
             first_iter = False
         else:
             arr.save(prefix_path=prefix_path, dataset=name, file_format=file_format, mode='append')
+
+
+def import_data(read_path: str, write_file: str = None,
+                return_obj: bool = True, index: bool = False):
+    """
+    Import data from Pandas to Arkouda readable format
+
+    Parameters
+    __________
+    read_path: str
+        path to file where pandas data is stored. This can be glob expression for parquet formats.
+    write_to: str, optional
```

Comment:
this doesn't match the parameter name in the function def (`write_file`)

Reply:
Updated